index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script  src="js/index.js"></script>
+    <meta charset="UTF-8">
+    <title>Nelson and Ahmed</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar" class="nav">
+      <ul class="nav-list">
+        <li>
+          <a href="#welcome-section">About</a>
+        </li>
+        <a id="profile-link" href="https://github.com/borrnelsonk" target="_blank" > Git Profile </a>  
+        </li>  
+        <li>
+          <a href="#projects">Work</a>
+        </li>
+        <li>
+          <a href="#contact">Contact</a>
+        </li>
+      </ul>
+    </nav>
+    
+
+ <section id="welcome-section" class="welcome-section">
+      <h1>Hello welcome to Ahmed and Nelsons page</h1>
+      <p>We are new web developers</p>
+    </section>
```

Comment:
Nice work but I think child elements should have an indentation of two with the space bar and the opening and closing parent element should be aligned to make the code more readable and clean.